### ISSUE TITLE
(Self-voicing) Gesture dialog needs "Close" label 

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#381]
@jessegreenberg, the dialogs are good.

While testing the dialogs, I noticed that the Back button on the Gestures dialog does not have a label that is spoken on focus.

I think it needs "Close" as the label in case a learner swipes back instead of forward when on that dialog.

**Comment by terracoda** [comment]
The rest of the dialogs seem to work fine, so I change the title of this issue.

**Comment by terracoda** [comment]
@jessegreenberg verified by demoing in zoom. Closing.
